src/mrinufft/io/nsp.py

```diff
@@ -69,6 +75,12 @@ def write_gradients(
         binary file, by default False
     final_positions : np.ndarray, optional
         Final positions. Shape (num_shots, dimension), by default None
+    start_skip_samples : int, optional
+        Number of samples to skip in ADC at start of each shot, by default 0
+        This works only for version >= 5.1.
```

Comment:
Versions of what ? Is it going to be clear to users ?

Reply:
Well versions of trajectory, which in this docstring corresponding to `write_trajectory`, they should have enough context from above, where `version` is a parameter.